rmgpy/reaction.py

```diff
@@ -787,6 +788,28 @@ def reverse_arrhenius_rate(self, k_forward, reverse_units, Tmin=None, Tmax=None)
         kr.fit_to_data(Tlist, klist, reverse_units, kf.T0.value_si)
         return kr
 
+    def reverse_surface_arrhenius_rate(self, k_forward, reverse_units, Tmin=None, Tmax=None):
+        """
+        Reverses the given k_forward, which must be an Arrhenius type.
```

Comment:
Docstring says it must be an Arrhenius type, but it should in fact be SurfaceArrhenius ?

Reply:
I just copied and pasted from the reverse_arrhenius_rate one.  I'll fix the comment